extensions/wikia/Helios/Client.class.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Wikia\Helios;
 use Wikia\Util\RequestId;
+use Wikia\Util\GlobalStateWrapper;
```

Comment:
Name spaced classes should live in `/lib/Wikia`. They aren't really PSR-* compliant under `/extensions`.